app/src/templates/checkboxes.html

```diff
@@ -0,0 +1,115 @@
+                                {% for boardgame in boardgames %}
+                                {% block elems %}
+                                {% endblock %}
+                                {% empty %}
+                                <div class="row">
+                                    <div class="col l12 m12 s12">
+                                        {% trans "Nothing to display" %}
+                                    </div>
+                                </div>
+                                {% endfor %}
+                            </div>
+                        </div>
+                    </div>
+                    {% block action %}
+                    <div class="card-action">
+                        <button type="submit" class="waves-effect waves-light btn">{% trans 'Submit' %}</button>
+                        <a href="{% url url_name %}" class="btn btn-default">Go Back</a>
+                    </div>
+                    {% endblock %}
+                </div>
+            </div>
+        </div>
+    </form>
+</div>
+{% endblock content %}
+
+
+{% block custom_scripts %}
+<script type="text/javascript">
+
+    const getNewInfo = () => {
+        var new_info = [];
+        var names = '';
+        $('input:checkbox').each(function () {
+            var $this = $(this);
+            if ($this.is(':checked')) {
+                var vote = $this.data('id');
+                new_info.push(vote);
+                var name = $this.data('name');
+                names += name + ", ";
+            }
+        })
+        names = names.slice(0, -2);
+        return [new_info, names];
+    }
+
+    const changeVote = (new_ids, new_names) => {
+        {% block info %}
+        {% endblock %}
+        $.ajax({
+            type: "POST",
+            dataType: 'json',
+            url: info_url,
+            data: {
+                'csrfmiddlewaretoken': '{{ csrf_token }}',
+                'new_ids': JSON.stringify(new_ids),
+            },
+            success: function () {
+                {% block message %}
+                {% endblock %}
+                M.toast({
+                    html: info_msg,
+                    displayLength: 2000,
+                    classes: "success"
+                });
+            },
+            error: function () {
+                M.toast({
+                    html: "Error setting boardgames votes.",
+                    displayLength: 2000,
+                    classes: "error"
+                });
+            }
+        });
+    }
+
+    $(document).ready(function () {
+        $("button").on('click', function (event) {
+            event.preventDefault();
+            var new_info = getNewInfo();
+            var new_ids = new_info[0];
+            var new_names = new_info[1];
+            console.log(new_ids, new_names);
+            changeVote(new_ids, new_names);
+        });
+    });
+
+</script>
+{% endblock custom_scripts %}
```

Comment:
Directory app/src/templates should be used for generic templates. If this file was written specifically for boardgames, then it should be moved to boardgames templates